VagrantProvision.sh

```diff
@@ -168,6 +179,7 @@ if  ! dpkg -l | grep --quiet google-chrome-stable; then
     if [ ! -f google-chrome-stable_current_amd64.deb ]; then
       wget --quiet https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
     fi
+    sudo apt-get -f -y -q install
```

Comment:
Because you so thoroughly documented the rationale behind the changes (which I really like, both the changes and the comments on the diff), what is the purpose of this install line both before and after?